assets/css/components/_cards.scss

```diff
@@ -1,65 +1,65 @@
 .homepage-cards {
-  display: grid;
-  grid-gap: rem(38);
-  grid-template-columns: repeat(1, 1fr);
-  padding: rem(89) rem(60) rem(50);
+	display: grid;
+	grid-gap: rem(38);
+	grid-template-columns: repeat(1, 1fr);
+	padding: rem(89) rem(60) rem(50);
```

Comment:
I'm not sure why this is, and it might just be something with GitHub. In some files the tab indentation appears bigger than in other file. Take this file where it looks like the length of 4 spaces, where as in the [_brand.scss](https://github.com/inclusive-design/wecount.inclusivedesign.ca/pull/194/files#diff-5218a6a3b1ecd3c57716791d2b8598f2) it looks more like 2 spaces. Any thoughts on what might be causing this and if it is something we can control.

Reply:
@jobara It looks the same to me— not sure what you are seeing?